src/Components/CreateImageWizard/formComponents/ReviewStep.js

```diff
@@ -112,6 +112,14 @@ const ReviewStep = () => {
             <TargetEnvOtherList />
           </TextContent>
         )}
+        {getState()?.values?.['target-environment']?.wsl && (
+          <TextContent>
+            <Text component={TextVariants.h3}>
+              Windows Subsystem for Linux (WSL)
```

Comment:
Should we put an extension in the parentheses so it's consistent with the other options?

```suggestion
              WSL - Windows Subsystem for Linux (.tar.gz)
```